modules/mono/mono_gd/gd_mono_android.cpp

```diff
@@ -81,7 +81,8 @@ struct ScopedLocalRef {
 	ScopedLocalRef &operator=(const ScopedLocalRef &) = delete;
 
 	ScopedLocalRef(JNIEnv *p_env, T p_local_ref) :
-			env(p_env), local_ref(p_local_ref) {
+			env(p_env),
+			local_ref(p_local_ref) {
```

Comment:
I think you included this change by accident.

Reply:
Yeah, I saw it but forgot about it. Interestingly, it's the only place where the autoformatter disagrees with the existing C++ code.

Reply:
Looks like that autoformatter needs to be taught a lesson.